src/pages/playlist.js

```diff
@@ -1,42 +1,109 @@
 import React, { useEffect, useState } from "react";
-import { showPlaylist } from '../api/hudl';
+import { showPlaylist } from "../api/hudl";
 import "./playlist.css";
 import logo from "../assets/logo.svg";
 import Media from "./media.js";
 import { useParams } from "react-router-dom";
 
 export default function Playlist() {
-  let [playlist, setPlaylist] = useState(null)
-  let [error, setError] = useState(null)
+  let [playlist, setPlaylist] = useState(null);
+  let [error, setError] = useState(null);
   let { playlist_id } = useParams();
 
   const fetchPlayistDetails = async () => {
     try {
       let response = await showPlaylist(playlist_id);
       setPlaylist(response.data.playlist);
     } catch (error) {
-      setError(error.message)
+      setError(error.message);
     }
-  }
+  };
 
   useEffect(() => {
-    fetchPlayistDetails()
-  }, [])
+    fetchPlayistDetails();
+  }, []);
 
   if (error === null && playlist === null) {
-    return <h3>loading...</h3>
+    return <h3>loading...</h3>;
   } else {
     return playlist !== null ? (
-      <div id="playlist" style={{ backgroundImage: `url("${playlist.user.cover_image.url}")` }}>
-        <center> <img id="logo" src={logo} alt="logo-image"></img></center>
+      <div
+        id="playlist"
+        style={{ backgroundImage: `url("${playlist.user.cover_image.url}")` }}
+      >
+        <center>
+          {" "}
+          <img id="logo" src={logo} alt="logo-image"></img>
+        </center>
         <h2 id="text">item Amazing Playlist</h2>
         <h5 id="text1">Created by {playlist.user.username}</h5>
+<<<<<<< Updated upstream
         <img id="profile" src={playlist.user.profile_picture.url} alt="profile-picture" />
         
+=======
+        <img
+          id="profile"
+          src={playlist.user.profile_picture.url}
+          alt="profile-picture"
+        />
+        <Media playlistId={playlist.id} />
+        <div id="Footer">
+          <svg
```

Comment:
Do no directly paste svg into react component
This makes the file extremely hard to read.

create a file in assets folder call it footer.svg
Paste that code in it, and then import it here